src/api/CustomerApi.ts

```diff
@@ -4,7 +4,7 @@ export const choosableColors = ['red', 'pink', 'rebeccapurple', 'grey'];
 
 export interface Customer {
   name: string;
-  color: string; // TODO: Create a new Color type, I only want the "choosableColors" above to be permitted on Customers
+  color: string; 
```

Comment:
remove this change, you made it in the task 1 branch, I'll show you how to combine them.